lib/utils/bedroomCalculator.ts

```diff
@@ -0,0 +1,60 @@
+function calculateBedrooms(...people: Array<any>) {
+  // one bedroom each...unless it's a couple
+  let relationshipType = people[0].pop();
+  let over16 = people.filter(([age, _]) => {
+    return age >= 16;
+  }).length;
+
+  // 0.5 bedroom each.
+  const under10 = people.filter(([age]) => age < 10).length / 2;
+
+  if (relationshipType === 'partner') {
+    over16 = over16 - 1;
```

Comment:
same here - try to define a new `const` rather than modifying a variable defined with `let`.